src/github.com/getlantern/flashlight/config/config.go

```diff
@@ -204,6 +206,10 @@ func (cfg *Config) ApplyDefaults() {
 		cfg.UIAddr = "localhost:16823"
 	}
 
+	if cfg.CloudConfig == "" {
+		cfg.CloudConfig = "https://s3.amazonaws.com/lantern_config/cloud.valencia.yaml.gz"
+	}
+
```

Comment:
I realized that we weren't defaulting the cloud config URL anywhere, so I added that.